vault/dendron.release.2021-11-09.md

```diff
@@ -0,0 +1,76 @@
+---
+id: EjPEcAkZIDgzHaVMk4teA
+title: Release Notes (version 0.68)
+desc: ''
+updated: 1636480628276
+created: 1636478484110
+---
+
+Dendron 0.68 has sprouted  🌱
+
+Take a look at the latest [Dendron wiki](https://wiki.dendron.so), and you'll see some new features directly from these highlights:
+
+* Local TOC in the right-hand nav for each page in the published docs
+* Newly combined search bar that can go directly to a note, or search note contents using the `?` prefix
+
+Also: an enhancement in this release delivers breaking changes announced last week, due to some renaming of commands, which can be referenced in the changelog.
+
+### Highlights
+* feat(schemas): adding new command - create schema from hierarchy
```

Comment:
can we add docs to any highlights/enhancements that have them? see comment above about release notes standing on its own